testing/test_pdb.py

```diff
@@ -1157,12 +1157,13 @@ def runcall(self, *args, **kwds):
     result = testdir.runpytest(
         str(p1), "--pdbcls=really.invalid:Value", syspathinsert=True
     )
-    result.stderr.fnmatch_lines(
+    result.stdout.fnmatch_lines(
         [
-            "ERROR: --pdbcls: could not import 'really.invalid:Value': No module named *really*"
+            "*= FAILURES =*",
+            "E * --pdbcls: could not import 'really.invalid:Value': No module named *really*",
         ]
     )
-    assert result.ret == 4
+    assert result.ret == 1
```

Comment:
Note that this delays the UsageError now, which is not handled too well by pytest (e.g. via exitcode).
IIRC UsageErrors are only handled during config parsing/setup.